src/clangd-context.ts

```diff
@@ -58,7 +58,7 @@ export class ClangdContext implements vscode.Disposable {
           await config.getSecureOrPrompt<string[]>('arguments', workspaceState),
       options: {cwd: vscode.workspace.rootPath || process.cwd()}
     };
-    const traceFile = config.get<string>('trace');
+    const traceFile = config.get<string>('trace', '');
```

Comment:
I think that echoing defaults already given in the package.json is unacceptable duplication (and causes practical problems e.g. someone changing the default in the wrong place).

Can we add a `!` assertion in config.get() instead?

Reply:
That's fair. I believe the defaults can be retrieved through `WorkspaceConfiguration.inspect()`, but that may be more trouble than it's worth. Realistically adding asserts here would only cause a problem if a default value was removed from `package.json`, but I expect that's quite unlikely.